index.html

```diff
@@ -478,6 +591,130 @@ <h5 class="card-header-text">Web Development</h5>
     </div>
         </section>
 
+        <section class="video">
+            <div class="container">
+                <div class="section-header" id="clients-header">
+                    <h2 class="section-header-title">We are providing best business service.</h2>
+                    <p class="section-header-text">Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics </p>
+                </div>
+                <div class="content-video">
+                    <video class="video" controls poster="img/cover-video.jpg">
+                        <source src="https://media.geeksforgeeks.org/wp-content/uploads/20200409094356/Placement100-_-GeeksforGeeks2.mp4" type="video/mp4">
+                    This video is not supported by your browser.
+                    </video>
+                    <div class="playpause"></div>
+                </div>
+            </div>
+        </section>
+
+        <section class="clients">
```

Comment:
Maybe the name "customer-feedback/ clients-feedback or just feedback" would be more suitable for this section